access/types/forms.py

```diff
@@ -452,6 +458,12 @@ def grade_field(self, i, configuration):
         else:
```

Comment:
GitHub does not let me comment on lines that were not modified. Anyway, in this `grade_field` method, the if statement with "table-radio" and "table-checkbox" uses the `self.grade_checkbox` method too. Its return value was changed, so the method call should be updated everywhere where the method is used.